src/ace/managers/sprite.c

```diff
@@ -23,14 +23,29 @@ typedef struct tSpriteChannel {
 
 static const tView *s_pView;
 static tSpriteChannel s_pChannelsData[HARDWARE_SPRITE_CHANNEL_COUNT];
-static const tHardwareSpriteHeader CHIP s_uBlankSprite;
+static UBYTE s_ubOwnsBlankSprite;
 static tCopBlock *s_pInitialClearCopBlock;
+ULONG *s_pBlankSprite;
 
 static void spriteChannelRequestCopperUpdate(tSpriteChannel *pChannel) {
 	pChannel->ubCopperRegenCount = 2; // for front/back buffers in raw mode
 }
 
-void spriteManagerCreate(const tView *pView, UWORD uwRawCopPos) {
+void spriteManagerCreate1(const tView *pView, UWORD uwRawCopPos) {
+	s_pBlankSprite = memAllocChipClear(sizeof(ULONG));
+	spriteManagerCreate2(pView, uwRawCopPos, s_pBlankSprite);
+	s_ubOwnsBlankSprite = 1;
+}
+
+void spriteManagerCreate2(const tView *pView, UWORD uwRawCopPos, ULONG *pBlankSprite) {
```

Comment:
how about only having this function and if pBlankSprite is null then allocate and set s_ubOwnsBlankSprite to 1?